tests/framework/db/oci/QueryBuilderTest.php

```
@@ -357,6 +358,192 @@ public function testExecuteResetSequenceForLegacyTriggerBackedTable(): void
+                <<<SQL
+                SELECT DEFAULT_ON_NULL FROM USER_TAB_COLUMNS
+                WHERE TABLE_NAME = 'reset_identity_by_default' AND COLUMN_NAME = 'id'
+                SQL
+            )->queryScalar(),
+            "The 'ON NULL' variant must not be gained.",
+        );
+    }
+
+    public function testExecuteResetSequencePreservesIdentityOptions(): void
+    {
+        $db = $this->getConnection();
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $db. Configured minimum length is 3. (undefined)

(ShortVariable)